protoc-gen-gogo/generator/generator.go

```diff
@@ -549,6 +548,19 @@ var uniquePackageName = make(map[*descriptor.FileDescriptorProto]string)
 // value is the name that appears in the generated code.
 var pkgNamesInUse = make(map[string][]*FileDescriptor)
 
+func pkgNameInUse(pkg string, f *FileDescriptor) bool {
```

Comment:
Seems this is only a cleanup of the old code.
Could you revert the use of this function so that the diff with golang/protobuf is as small as possible.